README.md

```diff
@@ -55,6 +55,22 @@ pytest
 
 To learn how to use `C-Star`, check out the [documentation](https://c-star.readthedocs.io/en/latest/index.html).
 
+# Environment Variables
+The following environment variables control C-Star's behavior:
+
+| Variable                     | Default                            | Effect                                                                                                                                                                                                            |
+|------------------------------|------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
unless you're using some formatting (centering, etc), you might use this to accomplish the same thing and avoid text wrap:

`|-|-|-|`